.gitignore

```diff
@@ -69,3 +69,4 @@ certs/*.pem
 
 # Database Content
 data/*
+strategy_v002.js
```

Comment:
Is this just a local file you had? Or do we expect this to be generated or something? If it was a one off, you can take it out of the .gitignore and just not commit this locally.